platform/windows/os_windows.cpp

```diff
@@ -1332,7 +1332,11 @@ void OS_Windows::vprint(const char* p_format, va_list p_list, bool p_stderr) {
 	if (p_stderr)
 		fwprintf(stderr,L"%s",wbuf);
```

Comment:
> Also, is the distinction necessary or would MSVC handle %S just as well?

No, doesn't work with MSVC, it won't even compile.

BTW, I think it's easier to read if an `#ifdef` is used instead of an `#ifndef`. Got confused the first time I saw.